sdk/CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 0.4.1 - 2023-07-DD
```

Comment:
Is this something you want released today? Otherwise let's make it RC0

Reply:
I don't need it released today